rotkehlchen/db/upgrades/v28_v29.py

```diff
@@ -99,13 +94,13 @@ def upgrade_v28_to_v29(db: 'DBHandler', progress_handler: 'DBUpgradeProgressHand
     PRIMARY KEY (address, xpub, derivation_path)
     );
     """)  # noqa: E501
-    cursor.executemany(
-        'INSERT INTO xpub_mappings(address, xpub, derivation_path, account_index, derived_index) '
+    write_cursor.executemany(
+        'INSERT INTO xpub_mappings(address, xpub, derivation_path, account_index, derived_index) '  # noqa: E501
         'VALUES(?, ?, ?, ?, ?);',
         xpub_mappings_data,
     )
     progress_handler.new_step()
-    cursor.execute("""
+    write_cursor.execute("""
```

Comment:
you are making way way too many changes by renaming cursor here. Makes it very difficult to review and make sure you did not fuck something up. Please don't do that. In your PRs touch as little things as possible. It's really hard to review.